Resources/public/js/views/ez-navigationhubview.js

```diff
@@ -492,75 +571,80 @@ YUI.add('ez-navigationhubview', function (Y) {
              */
             zones: {
                 value: {
-                    'create': {
-                        name: 'Create',
-                        hint: 'Creating & Editing',
-                        identifier: 'create',
-                    },
-                    'deliver': {
-                        name: 'Deliver',
-                        hint: 'Publishing & Engagin',
-                        identifier: 'deliver',
-                    },
-                    'optimize': {
-                        name: 'Optimize',
-                        hint: 'Statistics & Analytics',
-                        identifier: 'optimize',
-                    },
+                    'platform': 'Platform',
+                    'studio': 'Studio',
+                    'studioplus': 'Studio Plus',
+                    'admin': 'Admin. Panel',
```

Comment:
Think we can remove the `.`, not part of the screenshoot I have seen so far and as PM said that was not needed.